### PR TITLE
AC Test Fix - LRAC-11639 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-11639

Hey team, I'm changing the test that validated the DM functionality in the acceptance routine, because the previous test that is more complete is having problems because of this [bug](https://issues.liferay.com/browse/LRAC-11639).

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase [hunk 1]
```diff
@@ -1058,9 +1060,13 @@ definition {
 	@description = "Documents visitor behavior card shows expected amount of views"
 	@priority = "5"
 	test VisitorBehaviorCardShowsExpectedAmountOfPreviewsInDM {
-		property portal.acceptance = "true";
+
+		//property portal.acceptance = "true";
+
 		property portal.release = "true";
```
Comment (joshitagaki) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase:1066]: Thanks for finding this! Do you think we should also comment out the portal.release run as well?

Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase:1066]: Hey Josh, I've checked the last 4 runs of tests that we have from the portal.release of the new DXP U release and those tests are constantly passing there. I believe we can leave this property, but I'm wondering why the problem doesn't happen there 🤔 😆

**Review by christianstokes — APPROVED** [review]
Approved. Thank you @ViniLopes20

**Comment by christianstokes** [comment]
ci:forward

**Pull request closed without merging**